build.sh

```diff
@@ -1,9 +1,91 @@
-#!/usr/bin/env bash
+#!/bin/sh
```

Comment:
`#!/bin/sh` is not portable. Use `#!/usr/bin/env sh`

Also POSIX-compatible shell scripts are kind of awful :shrug:

Reply:
portability
what if somebody uses dash, fish or pdksh?
I mean yes its not as pretty but thats not very important

Reply:
`/bin/sh` is not guaranteed to exist, gonna change that.

Reply:
fixed.